hips/tiles/hipsdescription.py

```diff
@@ -0,0 +1,75 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HiPS description class -- provides methods for parsing the HiPS file properties."""
+
+__all__ = [
+    'HipsDescription',
+]
+
+from collections import OrderedDict
+
+
+class HipsDescription:
+    """This class provides methods for parsing the HiPS properties file.
```

Comment:
Docstrings should always be a summary line, followed by an empty line, followed by more description or sections like `Parameters` etc.
The pattern we use in Astropy is to document the parameters of `__init__` in the `Parameters` section of the class-level docstring (and to never put a docstring on `__init__`. So here, please add it here to describe the `properties` parameter.